frontend/corrections.py

```diff
@@ -182,7 +181,7 @@ def approve_correction(
                            email_template, corrected_val)
             return jsonify(
                 create_response_message(
-                    True, "Successfully updated hadith")
+                    True, "Successfully Updated Hadith")

```

Comment:
please revert to the original. It is a sentence, not a title.

Reply:
Updated